src/v/datalake/translation/tests/partition_translator_tests.cc

```diff
@@ -452,7 +452,7 @@ TEST_F_CORO(partition_translator_fixture, test_batching) {
 
     std::exception_ptr ex = nullptr;
     try {
-        RPTEST_REQUIRE_EVENTUALLY_CORO(10s, [&test_ctx]() {
+        RPTEST_REQUIRE_EVENTUALLY_CORO(20s, [&test_ctx]() {
```

Comment:
curious: what's going on here?

Reply:
something changed in bazel-dbg, still not clear (think those changes landed _after_ I merged my PR, so the test never ran in the failing config). Per logs, the debug build seems slower than the cases where it passed (it is pretty close to getting passed, so just bumped the timeout).

Reply:
cool cool. was wondering whether we might give a little more slack if the timeliness isn't part and parcel of the test, but what you're describing sgtm.